BridgeAppSDK/SBAStepProgressView.swift

```diff
@@ -121,8 +126,8 @@ open class SBAStepProgressView: UIView {
         progressView.translatesAutoresizingMaskIntoConstraints = false
         stepCountLabel.translatesAutoresizingMaskIntoConstraints = false
         
-        backgroundView.backgroundColor = UIColor.headerViewProgressBackground
-        progressView.backgroundColor = UIColor.headerViewProgressBar
+        backgroundView.backgroundColor = SBAStepProgressView.progressBackgroundColor
+        progressView.backgroundColor = SBAStepProgressView.progressColor
```

Comment:
Subclassing and overriding will not return the progress bar background color. For that, you need to do this instead:

`type(of: self).progressBackgroundColor`

Reply:
That's a good tip, will make the change

Reply:
fixed